app/src/main/cpp/skyline/gpu/interconnect/graphics_context.h

```diff
@@ -925,6 +941,34 @@ namespace skyline::gpu::interconnect {
                     }
                 }
 
+
+                if (!program.info.storage_buffers_descriptors.empty()) {
+                    descriptorSetWrites.push_back({
+                        .dstBinding = bindingIndex,
+                        .descriptorCount = static_cast<u32>(program.info.storage_buffers_descriptors.size()),
+                        .descriptorType = vk::DescriptorType::eStorageBuffer,
+                        .pBufferInfo = bufferInfo.data() + bufferInfo.size(),
+                    });
+
+                    for (auto &storageBuffer : program.info.storage_buffers_descriptors) {
+                        layoutBindings.push_back(vk::DescriptorSetLayoutBinding{
+                            .binding = bindingIndex++,
+                            .descriptorType = vk::DescriptorType::eStorageBuffer,
+                            .descriptorCount = 1,
+                            .stageFlags = pipelineStage.vkStage,
+                        });
+
+                        auto view{GetSSBOViewFromDescriptor(storageBuffer, pipelineStage.constantBuffers)};
+                        std::scoped_lock lock(view);
+                        bufferInfo.push_back(vk::DescriptorBufferInfo{
```

Comment:
{} not (), though mark might do this globally in a while so guess it's just a nit